src/content/guides/production.md

```diff
@@ -96,6 +98,7 @@ __webpack.prod.js__
 + const common = require('./webpack.common.js');
 +
 + module.exports = merge(common, {
++   mode: 'production',
 +   plugins: [
```

Comment:
this is not how you do it in v4, go for:

```
...,
optimization: {
  minimizer: [
    new UglifyJsPlugin({}),
    ...
  [
}
```

Reply:
Should it be removed?

Reply:
yes mode: production does minification now by default, providing a custom minimizer only lets you override default options

Reply:
Ok, I updated it